### DISCUSSION TITLE
Working with Collections

### DISCUSSION THREAD

**Discussion by apolyanov** [mikro-orm/mikro-orm#5970] [General]
Why can I use EntityManager.getReference() to remove items from a Collection but not Reference.createFromPK()?

**Comment by B4nan** [comment]
Because those references need to be managed (and `Reference.createFromPK()` is creating unmanaged references only, it's a static method without any context). The use case for `Reference.createFromPK()` (or `rel()` which is the same) is the entity constructor mainly.
